mkdocs.yml

```diff
@@ -0,0 +1,65 @@
+#
+# Copyright (c) 2021 - for information on the respective copyright owner
+# see the NOTICE file and/or the repository https://github.com/carbynestack/carbynestack.github.io.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+INHERIT: config/nav.yml
+
+site_name: Carbyne Stack
+site_url: https://carbynestack.io/
+site_author: Carbyne Stack Community
+site_description: >-
+  Carbyne Stack Documentation
+copyright: "Copyright © 2021 The Carbyne Stack Authors"
+
+repo_url: https://github.com/carbynestack/carbynestack.github.io
```

Comment:
Maybe you'd like to consider changing the linked repository to [https://github.com/carbynestack/carbynestack](https://github.com/carbynestack/carbynestack) as it is considered to be the umbrella repository for CarbyneStack